unleash-yggdrasil/src/lib.rs

```diff
@@ -404,9 +412,9 @@ impl EngineState {
                 return false;
             }
 
-            let parent_enabled = self.enabled(compiled_parent, context, &None); //parent toggles explicitly don't support custom strategies
+            let parent_enabled = self.enabled(compiled_parent, &context);
```

Comment:
We now explicitly allow custom strategies for parent toggles. I think this is probably okay, but feel free to sanity check me here